test/pages/HaveYouUsedMethodOneForSimilarGoodsInPastPageSpec.scala

```diff
@@ -28,4 +31,18 @@ class HaveYouUsedMethodOneForSimilarGoodsInPastPageSpec extends PageBehaviours {
 
     beRemovable[Boolean](HaveYouUsedMethodOneForSimilarGoodsInPastPage)
   }
+  "cleanup" - {
+    "should reset DescribeTheSimilarGoodsPage" - {
+      "when HaveYouUsedMethodOneForSimilarGoodsInPastPage is changed to No" in {
+        val emptyUserAnswers = UserAnswers("id", DraftId(1))
+
+        val ua = emptyUserAnswers
+          .set(DescribeTheSimilarGoodsPage, "similarities")
+          .get
+
+        HaveYouUsedMethodOneForSimilarGoodsInPastPage.cleanup(Some(false), ua) mustBe
+          Success(emptyUserAnswers)
+      }
+    }
+  }
```

Comment:
We want to be testing all scenarios here, namely what happens when the cleanup is called with a None, and Some(true).